spec/fastsync/version2/scheduler.tla

```diff
@@ -0,0 +1,532 @@
+------------------------------- MODULE fsv2_scheduler ---------------------------------------------
+(*
+ A specification of the fast sync scheduler that is introduced in blockchain/v2:
+ 
+ https://github.com/tendermint/tendermint/blockchain/v2
+ 
+ The model includes:
+    - a scheduler that maintains the peers and blocks that it receives from the peers, and
+    - one environment simulating a correct peer
+    
+ This specification focuses on the events that are received and produced by the scheduler.
+ Communication between the scheduler and the other fastsync components is not specified.
+*)
+
+EXTENDS Integers, FiniteSets, TLC
```

Comment:
It looks like you are not using the operators from the TLC module

Reply:
i used some Prints from there earlier, removed now.